pandemic51/core/events.py

```diff
@@ -94,10 +94,15 @@ def add_events_to_points(points, events):
 
 
 def _find_event_index(time, event_times):
-    idx = np.argmin(np.abs(time - event_times))
+    events = np.asarray([_mktime(t) for t in event_times])
```

Comment:
you can just fix this above in lines 82 and 85

Reply:
But then the conditional on line 100 does not work.

Reply:
oh okay. Maybe switching to `np.abs(time - events).total_seconds()` is cleaner then?

I may not understand what the problem is. In any case, carry on!